From c9df77b21e6504e5fbeea909518e962d28249608 Mon Sep 17 00:00:00 2001
From: Jin Wei <wei.a.jin@intel.com>
Date: Thu, 21 Mar 2013 14:12:15 +0800
Subject: [PATCH 2/2] Enable GUI for PDK

We have to lunch the surfaceflinger in init.rc not by system_init

Change-Id: Ifd12f3c41bbef65c1e5084a64d550c91e25b0eca
Signed-off-by: Jin Wei <wei.a.jin@intel.com>
---
 init.rc |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/init.rc b/init.rc
index eac885b..b9e76b0 100644
--- a/init.rc
+++ b/init.rc
@@ -350,7 +350,7 @@ on boot
     setprop net.tcp.buffersize.evdo    4094,87380,262144,4096,16384,262144
 
 # Set this property so surfaceflinger is not started by system_init
-    setprop system_init.startsurfaceflinger 1
+    setprop system_init.startsurfaceflinger 0
 
     class_start core
     class_start main
@@ -445,11 +445,11 @@ service debuggerd /system/bin/debuggerd
 #    user root
 #    group radio cache inet misc audio log
 
-#service surfaceflinger /system/bin/surfaceflinger
-#   class main
-#   user system
+service surfaceflinger /system/bin/surfaceflinger
+   class main
+   user system
 #   group graphics drmrpc
-#   onrestart restart zygote
+   onrestart restart zygote
 
 service zygote /system/bin/app_process -Xzygote /system/bin --zygote --start-system-server
     class main
-- 
1.7.9.5

